web/src/components/metrics/metrics-content.tsx

```diff
@@ -231,20 +201,15 @@ export const MetricsContent: React.FC<{
             }}
           >
             <ChartAxis fixLabelOverlap />
-            <ChartAxis
-              dependentAxis
-              showGrid
-              fixLabelOverlap
-              tickFormat={y => getMetricValue(y, metricFunction, metricType)}
-            />
+            <ChartAxis dependentAxis showGrid fixLabelOverlap tickFormat={y => getMetricValue(y, metricType, 'avg')} />
```

Comment:
Anything but 'sum' would work (and we don't want 'sum' in time-series representations). It's always "Bps" or "packets/sec" here.

In fact, instead of calling `getMetricValue` we could directly call `metricType === 'bytes' ? bytesPerSeconds(y) : elementPerMinText(y)` ; or do a proper switch for that; I will change that.

Reply:
done in 5c8bbfb4413713dfb9c4c86cb6babc7f479dd364 (now explicitly referring to rate display, not mentioning 'avg')

Reply:
Sorry, wrong commit, here it is: https://github.com/netobserv/network-observability-console-plugin/pull/195/commits/4b1f878c6358fff0dd7f07d80a4c4155fab2e1ad